src/main/java/reposense/git/GitShow.java

```diff
@@ -48,7 +50,11 @@ public static CommitHash getExpandedCommitHash(String root, String shortCommitHa
     }
 
     /**
-     * Returns date of commit associated with commit hash.
+     * Returns {@code Date} of commit associated with commit hash.
+     *
+     * @throws CommitNotFoundException if no commit exists for the given {@code commitHash}.
+     * @throws ParseException if the date string for the given {@code commitHash} could not be parsed into
+     * a {@code Date}.
```

Comment:
I think it can be specified as 'could not be parsed into a {@code Date} _object_.' in order to highlight the difference between that and the date string.

Reply:
Fixed